### DISCUSSION TITLE
BOJ 24913 개표 문제 질문

### DISCUSSION THREAD

**Discussion by jusung-c** [ghdcksgml1/Algorithm_Study#184] [질문과 답변] [answered]
문제 : https://www.acmicpc.net/problem/24913

질문 : http://colorscripter.com/s/WIbmEgh

하다가 안돼서 다른 사람 풀이 보고 이해를 했는데
뭔가 놓치고 있는 게 있나봐요.. 자꾸 41퍼에서 막히네오ㅠ
질문에 올라와있는 반례는 다 통과해요우..

맞왜틀 ㅠㅠㅠㅠ

**Comment by ghdcksgml1** [comment, marked as answer]
저도 10점입니다. ㅎㅎ..

**Reply by jusung-c** [reply]
범위를 무지성으로 전부 long으로 바꿔주니 풀리네여 ㅋㅋㅋㅋㅋ
왜 풀리는지는 모르겠다요

**Reply by BanApp** [reply]
오ㅋㅋㅋㅋ 근데 이거 봐도 뭔소린지 이해를 못하겠.. 국어를 다시 공부해야하나 봅니다..

**Reply by jusung-c** [reply]
ㅇㅈ..
